qmf/src/plugins/messageservices/imap/imapservice.cpp

```diff
@@ -86,7 +86,9 @@ QString connectionSettings(ImapConfiguration &config)
     result << config.mailServer();
     result << QString::number(config.mailPort());
     result << QString::number(config.mailEncryption());
+#ifndef USE_ACCOUNTS_QT
     result << QString::number(config.mailAuthentication());
+#endif
```

Comment:
We detect this automatically, so no need to check anymore here since it can change when caps are requested.